### PR TITLE
New page about allowlisting Elastic Endpoint in 3rd-party AV software

### PR DESCRIPTION
**Pull request by benironside**
Addresses #3535 by adding a new page with information about how to allowlist Elastic Endpoint in 3rd-party AV software on different OSes.

Preview: [Allowlist Elastic Endpoint ](https://security-docs_bk_4439.docs-preview.app.elstc.co/guide/en/security/master/allowlist-endpoint-3rd-party-av-apps.html)

### CODE REVIEW

File: docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+NOTE: Your AV software may refer to allowlisted processes as process exclusions, ignored processes, or trusted processes.
+
+NOTE: We recommend you allowlist both the file paths and digital signatures, if applicable. 
+
+[discrete]
+== Allowlist {elastic-endpoint} on Windows
+
+File paths:
+
+* ELAM driver: `c:\Windows\system32\drivers\elastic-endpoint-driver.sys (ELAM driver)`
```
Comment (ferullo) [inline docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc:15]:
```suggestion
* ELAM driver: `c:\Windows\system32\drivers\elastic-endpoint-driver.sys`
```

File: docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+File paths:
+
+* System extension (recursive directory structure): `/Applications/ElasticEndpoint.app/`
++
+NOTE: The system extension runs as `co.elastic.systemextension`.
+
+* Executable: `/Library/Elastic/Endpoint/elastic-endpoint`
++
+NOTE: The executable runs as `elastic-endpoint.exe`.
```
Comment (ferullo) [inline docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc:39]:
```suggestion
NOTE: The executable runs as `elastic-endpoint`.
```

File: docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+[discrete]
+== Allowlist {elastic-endpoint} on macOS
+
+File paths:
+
+* System extension (recursive directory structure): `/Applications/ElasticEndpoint.app/`
++
+NOTE: The system extension runs as `co.elastic.systemextension`.
+
+* Executable: `/Library/Elastic/Endpoint/elastic-endpoint`
```
Comment (ferullo) [inline docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc:37]: @ricardoungureanu should the app path be mentioned as well?

**Review by ferullo — CHANGES_REQUESTED** [review]
Minor changes requested. Thanks for doing this @benironside !

File: docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc (hunk 3 above)
Comment (ricardoungureanu) [reply docs/management/admin/allowlist-endpoint-3rd-party-av.asciidoc:37]:
```suggestion
* Executable: `/Library/Elastic/Endpoint/elastic-endpoint.app/Contents/MacOS/elastic-endpoint`
```
`/Library/Elastic/Endpoint/elastic-endpoint` is just a symlink

**Review by gabriellandau — APPROVED** [review]
No blockers, but since we're adding this to the regular docs, it would be good to bring [some version of this important point](https://github.com/elastic/endpoint/blob/0f5ac2e4f37a8fdf1c28dee26212c25d15d30a6d/PerformanceIssues-Windows.md#trusting-elastic-defend-in-other-software) along from the GH doc:

> It is important to note that file-, folder-, and path-based exclusions/exceptions are distinct from Trusted Applications and will NOT achieve the same result. The goal here is to ignore actions taken BY a process, not ignore the file that the process was spawned from. Files are different from processes.

Without it, we'll have users creating pointless file exceptions without achieving their desired results.

**Comment by ferullo** [comment]
This document can be backported to 8.0+ with the requested changes.

If the original macOS Endpoint path is used (i.e. `/Library/Elastic/Endpoint/elastic-endpoint`) this can be backported to 7.17 as well.

@gabriellandau @ricardoungureanu please contradict me if you disagree.

**Review by nastasha-solomon — COMMENTED** [review]
Just a handful of suggestions and minor corrections. LGTM otherwise!

**Review by gabriellandau — APPROVED** [review]
Thanks!

**Review by natasha-moore-elastic — APPROVED** [review]
LGTM!

**Pull request merged**